languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -43735,6 +43743,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
                 <example correction="García Márquez"><marker>Garcìa Mraquez</marker>.</example>
                 <example correction="Lluís Llach"><marker>Lluís Lach</marker>.</example>
                 <example correction="SÃO PAULO"><marker>SAO PAULO</marker></example>
+                <example correction="The Guardian">Ele lê o jornal <marker>the guardian</marker> toda manhã.</example>
+                <example>Ele disse the guardian is in the garden.</example>
```

Comment:
Maybe this test won't pass when testing PT alone because the 'ignore English' feature needs the English dictionary.

Reply:
I tested this one, and the behaviour here should be the same for `NULL` and `_english_ignore_`. There were _some_ examples I could indeed not add, as they would break in the absence of English.